### PR TITLE
[CMM-515] Add new icons: answers and profile_settings

### PR DESCRIPTION
**Pull request by reod**
Icons created by Design Core Team to include in style-guide - they are needed for Brainly Tools  (new moderation tools, no access yet - project not released).

new_icons

Jira task: https://brainly.atlassian.net/browse/CMM-515

### CODE REVIEW

File: src/images/icons/answers.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Redirect" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><path d="M322.3,83.68V73.85a34.45,34.45,0,0,0-34.48-34.46H76.13A34.45,34.45,0,0,0,41.68,73.85V290.46a34.47,34.47,0,0,0,34.45,34.46H86V147.68a64,64,0,0,1,64-64Z"/><path d="M396.13,157.54v-9.86a34.46,34.46,0,0,0-34.45-34.45H150a34.46,34.46,0,0,0-34.47,34.45V364.3A34.47,34.47,0,0,0,150,398.77h10.18V221.54a64,64,0,0,1,64-64Z"/><path d="M434.69,186.53H225.34a35.64,35.64,0,0,0-35.64,35.64V437a35.64,35.64,0,0,0,35.64,35.63H434.69A35.63,35.63,0,0,0,470.32,437V222.17A35.64,35.64,0,0,0,434.69,186.53ZM325.09,412.41l-1.55.08h-82a17.32,17.32,0,0,1-17.41-17.23,16.56,16.56,0,0,1,.18-2.46,17.33,17.33,0,0,1,15.55-14.69l1.55-.08h82.09a17.3,17.3,0,0,1,17.1,14.77,16.56,16.56,0,0,1,.18,2.46A17.29,17.29,0,0,1,325.09,412.41ZM425.68,342a18,18,0,0,1-7.46,1.61H241.81a17.24,17.24,0,1,1,0-34.47H418.22a17.81,17.81,0,0,1,7.46,1.62,17.07,17.07,0,0,1,0,31.24Zm0-68.92a18,18,0,0,1-7.46,1.61H241.81a17.24,17.24,0,1,1,0-34.47H418.22a17.81,17.81,0,0,1,7.46,1.62,17.07,17.07,0,0,1,0,31.24Z"/></svg>
```
Comment (coderitual) [inline src/images/icons/answers.svg:1]: hey @reod , please remove id from svg (it applies to the other svg as well)

**Comment by coderitual** [comment]
@reod could you please also add those icons in appropriate icon group inside `get-icon-group.js` file (answer -> essentials, profile_settings -> social).

File: src/images/icons/answers.svg (hunk 1 above)
Comment (reod) [reply src/images/icons/answers.svg]: Fixed.

**Comment by reod** [comment]
> @reod could you please also add those icons in appropriate icon group inside `get-icon-group.js` file (answer -> essentials, profile_settings -> social).
They were added there since the beginning :D

> Please add correct titles to the svg icon files as well. Thanks!
Fixed.

Thx!

**Review by coderitual — APPROVED** [review]

**Pull request merged**